### DISCUSSION TITLE
What is the default command line used for ffmpeg?

### DISCUSSION THREAD

**Discussion by ghost** [mrlt8/docker-wyze-bridge#957] [General]
What are the default parameters used by `ffmpeg` in the `wyze-bridge` configuration, so I can determine what to keep when making a custom `ffmpeg` command line in the `docker-compose.yml` file? I'd like to be able to add some text to the stream without having to spin up another instance of `ffmpeg` to have to reencode everything a second time.

**Comment by mrlt8** [comment]
setting `FFMPEG_LOGLEVEL=info` should log the ffmpeg command that is being used, but you basically just need something like this:

```yaml
- FFMPEG_CMD=-f h264 -i - -f rtsp -rtsp_transport tcp rtsp://0.0.0.0:8554/{cam_name}
```

If you have audio enabled, you'll need to read from the named pipe:

```yaml
- FFMPEG_CMD=-f h264 -i - -i /tmp/{cam_name}.wav -f rtsp -rtsp_transport tcp rtsp://0.0.0.0:8554/{cam_name}
```

**Comment by ghost** [comment]
Not working, unfortunately. What I need to be able to add some text to the camera output with the following option (assuming I have this right, timestamp the text of `ffmpegtext.txt` from the container's root folder onto the output):

Oh, and I have an nvidia GPU so h264_nvenc is enabled in the docker-compose.yml file.

```
-vf \"drawtext=textfile=\/ffmpegtext.txt:x=10:y=H-th-10:fontsize=12:fontcolor=white:shadowcolor=black:shadowx=5:shadowy=5\"
```
